datastore/tests/unit/test_client.py

```diff
@@ -353,7 +353,7 @@ def test_get_multi_miss(self):
         read_options = datastore_pb2.ReadOptions()
         ds_api.lookup.assert_called_once_with(
             self.PROJECT,
-            [key.to_protobuf()],
+            keys=[key.to_protobuf()],
```

Comment:
Ditto here (and throughout this file), please drop the `keys=`.

(Maybe you got it here as an artifact of a bad merge?)